tests/baselines/reference/commonJsExportTypeDeclarationError.errors.txt

```diff
@@ -0,0 +1,31 @@
+tests/cases/compiler/types1.ts(2,17): error TS1110: Type expected.
+tests/cases/compiler/types1.ts(3,1): error TS1005: '=' expected.
+tests/cases/compiler/types2.ts(2,19): error TS1110: Type expected.
+tests/cases/compiler/types3.ts(2,13): error TS2456: Type alias 'test' circularly references itself.
+
+
+==== tests/cases/compiler/test.js (0 errors) ====
+    module.exports = {
+       message: ""
+    }
+    
+==== tests/cases/compiler/types1.ts (2 errors) ====
+    import test from "./test";
+    export type test
+                    

```

Comment:
any idea what the error spans are here? (2,17) looks like it's *after* the last character of this line. Maybe the span needs to be adjusted in this case

Reply:
@sandersn It seems for this case TS parser throws an error on the next `line(3,1): error TS1005: '=' expected.`. https://www.typescriptlang.org/play?#code/C4TwDgpgBMEM7ALACgVA

Reply:
For `Type expected.` as well? For that, I thought the span should cover `test` in `export type test`.

Reply:
Yes.

```ts
// @Filename: ./types1.ts
import test from "./test";
export type test

// @Filename: ./types4.ts
type test
```

```ts
tests/cases/compiler/types1.ts(2,17): error TS1110: Type expected.
tests/cases/compiler/types1.ts(3,1): error TS1005: '=' expected.
tests/cases/compiler/types4.ts(1,10): error TS1110: Type expected.
tests/cases/compiler/types4.ts(2,1): error TS1005: '=' expected.

    
==== tests/cases/compiler/types1.ts (2 errors) ====
    import test from "./test";
    export type test
                    
!!! error TS1110: Type expected.
    
    
!!! error TS1005: '=' expected.
==== tests/cases/compiler/types4.ts (2 errors) ====
    type test
             
!!! error TS1110: Type expected.
    
    
!!! error TS1005: '=' expected.
```